src/Host/Client/Impl/Host/NullRHostConnector.cs

```diff
@@ -2,13 +2,16 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Globalization;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Common.Core;
 
 namespace Microsoft.R.Host.Client.Host {
     internal sealed class NullRHostConnector : IRHostConnector {
-        private static Task<RHost> Result { get; } = TaskUtilities.CreateCanceled<RHost>(new RHostDisconnectedException());
+        private static Task<RHost> Result { get; } = TaskUtilities.CreateCanceled<RHost>(
+            new RHostDisconnectedException(
+                string.Format(CultureInfo.InvariantCulture, Resources.NoConnectionsAvailable, Environment.NewLine, Environment.NewLine)));
```

Comment:
We have `xml:space="preserve"` - so it should just work if you type the message on several lines.

---
In reply to: [77705742](https://github.com/Microsoft/RTVS/pull/2246#discussion_r77705742) [](ancestors = 77705742)